alchemy-web/docs/providers/supabase/function.md

```diff
@@ -0,0 +1,220 @@
+- **`accessToken`** (`Secret`): Supabase access token. Falls back to `SUPABASE_ACCESS_TOKEN` environment variable.
+- **`baseUrl`** (`string`): Base URL for Supabase API. Default: `https://api.supabase.com/v1`.
+
+## Resource Properties
+
+The function resource exposes the following properties:
+
+- **`id`** (`string`): Unique identifier for the function
+- **`slug`** (`string`): URL-friendly identifier for the function
+- **`name`** (`string`): Function name
+- **`status`** (`string`): Current function status (e.g., "ACTIVE", "INACTIVE")
+- **`version`** (`number`): Current version number of the function
+- **`createdAt`** (`string`): ISO timestamp when the function was created
+- **`updatedAt`** (`string`): ISO timestamp when the function was last updated
+
+## Examples
+
+### Basic Function
+
+```typescript
+const helloFunction = Function("hello", {
+  projectRef: "my-project-ref",
+  body: `
+    export default function handler(req: Request) {
+      const { name } = await req.json();
+      return new Response(\`Hello \${name || 'World'}!\`);
+    }
+  `,
+});
+```
+
+### Function with JWT Verification
+
+```typescript
+const protectedFunction = Function("protected-api", {
+  projectRef: "my-project-ref",
+  verifyJwt: true,
+  body: `
+    export default function handler(req: Request) {
+      // JWT is automatically verified by Supabase
+      const authHeader = req.headers.get('Authorization');
+      return new Response('Authenticated request processed');
+    }
+  `,
+});
+```
+
+### Function with Import Map
+
+```typescript
+const advancedFunction = Function("advanced-api", {
+  projectRef: "my-project-ref",
+  importMap: {
+    "std/": "https://deno.land/std@0.168.0/",
+    "supabase/": "https://esm.sh/@supabase/supabase-js@2"
+  },
+  body: `
+    import { serve } from "std/http/server.ts";
+    import { createClient } from "supabase/mod.ts";
+    
+    serve(async (req) => {
+      const supabase = createClient(
+        Deno.env.get('SUPABASE_URL')!,
+        Deno.env.get('SUPABASE_ANON_KEY')!
+      );
+      
+      const { data } = await supabase.from('users').select('*');
+      return new Response(JSON.stringify(data));
+    });
+  `,
+});
+```
+
+### Function with Adoption
+
+```typescript
+// This will adopt an existing function if one with the same name already exists
+const existingFunction = Function("existing-function", {
+  projectRef: "my-project-ref",
+  adopt: true,
+  body: `
+    export default function handler(req: Request) {
+      return new Response("Updated function");
+    }
+  `,
+});
+```
+
+### Function that Won't be Deleted
+
+```typescript
+const persistentFunction = Function("persistent-function", {
+  projectRef: "my-project-ref",
+  delete: false, // Function will not be deleted when resource is destroyed
+  body: `
+    export default function handler(req: Request) {
+      return new Response("This function persists");
+    }
+  `,
+});
+```
+
+## API Operations
```

Comment:
None of the docs should list out the api operations. Just focus on the resources and how it's used in the context of alchemy